README.md

```diff
@@ -1 +1,31 @@
-# hw01
\ No newline at end of file
+
+
+> # "Anyone Can Cook""
+> ## - Chef Gusteau 
+
+![Chef Gusteau and Remy](https://user-images.githubusercontent.com/65682542/85363397-749ac800-b4e6-11ea-8ffb-5eab32255b88.jpg)
+
+
+# My name is Jack Ogle and I have been rewatching all the pixar movies during quarentine!
+
+## _*This is me*_
+!["I think I see a bald eagle!!!!"](https://user-images.githubusercontent.com/65682542/85432720-f9b6c900-b548-11ea-9c72-814d2e96f73c.jpg)
```

Comment:
I recommend leaving a blank line between different Markdown elements, such as between headers and paragraph text. It avoids rendering issues in more complex documents